src/modules/75-ci/components/PipelineStudio/BuildInfraSpecifications/BuildInfraSpecifications.tsx

```diff
@@ -442,6 +457,14 @@ export default function BuildInfraSpecifications({ children }: React.PropsWithCh
           const filteredLabels = getMapValues(
             Array.isArray(values.labels) ? values.labels.filter((val: any) => testLabelKey(val.key)) : values.labels
           )
+          const filteredTolerations = Array.isArray(values.tolerations)
+            ? values.tolerations.map((val: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232285)
```suggestion
            ? values.tolerations.map((val: unknown) => {
```